README.md

```diff
@@ -1,158 +1,132 @@
-EMMO - Python API for the European Materials & Modelling Ontology
-=================================================================
+# EMMO - Python API for the European Materials & Modelling Ontology
+
+<!-- markdownlint-disable MD033 -->
 
 ![CI tests](https://github.com/emmo-repo/EMMO-python/workflows/CI%20Tests/badge.svg)
 [![PyPI version](https://badge.fury.io/py/EMMO.svg)](https://badge.fury.io/py/EMMO)
 
+This package is based on [Owlready2] and provides an intuitive representation of [EMMO] in Python.
+It is available on [GitHub][EMMOntoPy] and on [PyPI][PyPI:EMMO] under the open source [BSD 3-Clause license](LICENSE.txt).
+
+The European Materials & Modelling Ontology (EMMO) is an ongoing effort to create an ontology that takes into account fundamental concepts of physics, chemistry and materials science and is designed to pave the road for semantic interoperability.
```

Comment:
Elemental Multiperspective Material Ontology

Reply:
Yes, this should be changed everywhere, here I have not changed content, only very minor in specific cases. But I can do a commit that goes through the entire documentation and updates this, but it can be done in a dedicated PR as well.

Reply:
I've created an issue for this to track the update of this: #230